dlp/acceptance/quickstart_test.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google, Inc
+# Copyright 2020 Google, Inc
```

Comment:
Please use `Copyright 2020 Google LLC` (note also no comma) for this and all other files.